presto-docs/src/main/sphinx/release/release-0.183.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.183
+=============
+
+General Changes
+---------------
+
+* Fix planning failure for queries that use ``GROUPING`` and contain aggregation expressions
+  that require implicit coercions.
+* Fix memory over-accounting for stages where aggregation is followed by other non-trivial work.
+* Fix formatting in EXPLAIN ANALYZE output.
+* Fix planning error for `LEFT JOIN` converted to `INNER JOIN` without equi-conditions.
+* Improve error message when query contains unsupported correlated subquery.
+* Improve performance of ``CAST(json_parse(...) AS ...)``
+* Add :func:`map_from_entries` and :func:`map_entries` function
+* Make possible for `CHAR` functions (like `subst`) to  have empty spaces at the end of
+  the result.
+* Spill to disk for aggregation uses revocable memory. Now spilling takes place only if
+  query runs out of cluster memory during execution.
+* Remove `experimental.operator-memory-limit-before-spill` config property and
+  `operator_memory_limit_before_spill` session property.
+* Add `experimental.aggregation-operator-unspill-memory-limit` config property and
+  `aggregation_operator_unspill_memory_limit` session property. Properties specify how much
+  memory can be used for merging spilled aggregation data from disk.
+
+Hive Changes
+------------
+
+* Work around `ORC-222 <https://issues.apache.org/jira/browse/ORC-222>`_ which results in invalid
+  summary statistics in ORC files when the input data contains invalid string data.
+  Normally when this occurs the query fails, but in rare situations incorrect results
+  may be returned.
+* Fix memory under-accounting for data read from ORC when a column contains string data.
+* Improve error message for small ORC files that are completely corrupt or not actually ORC.
+* Generate splits that match a predicate on hidden column "$path".
+
+TPCH Changes
+------------
+ * extend data statistics with column statistics for schemas `tiny` (`sf0.01`) and `sf1`
+
+TPCDS Changes
+-------------
+ * expose data statistics for schemas `tiny` (`sf0.01`) and `sf1`
+
+Web UI Changes
```

Comment:
We should write this out words rather than listing variable names. How about
```
* Add output rows, output size, written rows and written size to query detail page.